tutorials/export/exporting_for_android.rst

```diff
@@ -64,6 +64,11 @@ the JDK can be used for this purpose::
 
 This will create a ``debug.keystore`` file in your current directory. You should move it to a memorable location such as ``%USERPROFILE%\.android\``, because you will need its location in a later step. For more information on ``keytool`` usage, see `this Q&A article <https://godotengine.org/qa/21349/jdk-android-file-missing>`__.
 
+.. note::
+   It is important that the password is the same for the keystore and the key. This is a `known Android
```

Comment:
There needs to be an empty line after the note command.

Reply:
Done